test/factory/users_factory.js

```diff
@@ -13,7 +13,8 @@ const buildDefaultAttrs = params => ({
   ...params,
   email: params.email || generateEmail(),
   password: params.password || factory.chance('string', { length: 10 }),
-  points: Number(params.points) || 0
+  points: Number(params.points) || 0,
+  sessionsExpiredAt: params.sessionsExpiredAt || null
```

Comment:
If the default value is null, could we leave the `undefined` if it's not present in params?

Reply:
For some reason I can not delete this line 😕 . Because when I remove this line the factory generates random dates for `sessionsExpiredAt`